dhis-2/dhis-api/src/main/java/org/hisp/dhis/analytics/AnalyticsMetaDataKey.java

```diff
@@ -36,11 +36,16 @@ public enum AnalyticsMetaDataKey {
   ITEMS("items"),
   DIMENSIONS("dimensions"),
   PAGER("pager"),
+  ORG_UNITS("organisationUnits"),
   ORG_UNIT_HIERARCHY("ouHierarchy"),
   ORG_UNIT_NAME_HIERARCHY("ouNameHierarchy"),
-  ORG_UNIT_ANCESTORS("ouAncestors");
+  ORG_UNIT_ANCESTORS("ouAncestors"),
+  USER_ORGUNIT("USER_ORGUNIT"),
```

Comment:
does it need to have a different case respect the other keys ? Couldn't it be `userOrgUnit` ?